### DISCUSSION TITLE
running deploy.yml error

### DISCUSSION THREAD

**Discussion by gitfoxvm** [aristanetworks/avd#3537] [Q&A] [answered]
Hello, I'm getting the error below when running deploy.yml

ERROR! Python library 'jsonschema' version running 4.21.1 - requirement is jsonschema<4.18,>=4.5.1

**Comment by carlbuchmann** [comment, marked as answer]
@gitfoxvm, you will need to install the correct required version for `jsonshema`.
Please follow the following instruction to install the correct Python requirements for AVD: https://avd.arista.com/4.5/docs/installation/collection-installation.html#python-requirements-installation

**Reply by gitfoxvm** [reply]
Yes, I just did " pip install jsonschema==4.17.0" and it worked.  

I'm seeing these errors now:

TASK [arista.avd.eos_config_deploy_eapi : Replace configuration with intended configuration] **********************************************************************************************************************
fatal: [labspine2]: FAILED! => {"changed": false, "msg": "path specified in src not found"}
fatal: [labspine1]: FAILED! => {"changed": false, "msg": "path specified in src not found"}
fatal: [lableaf1-sw1]: FAILED! => {"changed": false, "msg": "path specified in src not found"}
fatal: [lableaf1-sw2]: FAILED! => {"changed": false, "msg": "path specified in src not found"}

**Reply by carlbuchmann** [reply]
@gitfoxvm By default we expected the intended configuration to be sourced from:

```yaml
{{ inventory_dir }}/intended/configs/{{ inventory_hostname }}.cfg
```

reference:
https://avd.arista.com/4.5/roles/eos_config_deploy_eapi/index.html#default-variables
https://github.com/aristanetworks/avd/blob/7e1ae930bf7721b0e22098c979ec11d40dcf910b/ansible_collections/arista/avd/roles/eos_config_deploy_eapi/tasks/main.yml#L30C31-L30C61
